test/evergreen.yml

```diff
@@ -130,6 +130,83 @@ functions:
               --enable-strict --enable-diagnostic --with-builtins=lz4,snappy,zlib
           fi
     - *make_wiredtiger
+  "compile wiredtiger docs":
+    - command: shell.exec
+      params:
+        working_dir: "wiredtiger/build_posix"
+        shell: bash
+        script: |
+          set -o errexit
+          set -o verbose
+
+          if [[ "${branch_name}" != "develop" ]]; then
+            echo "We only run documentation update task on the WiredTiger (develop) Evergreen project."
+            exit 0
+          fi
+
+          # Go through each supported branch to build the docs
```

Comment:
I think this is confusing with the line above where it implies we only want to run on develop. I think the above `branch_name` is the project name and the `branch` below is the WT github branch name. But they're so similar they are confusing. Add more explanation here.